services/QuillLMS/app/models/pdf_subscription.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: pdf_subscriptions
+#
+#  id                :bigint           not null, primary key
+#  filter_selections :jsonb
+#  frequency         :string           not null
+#  title             :string           not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  user_id           :bigint           not null
+#
+# Indexes
+#
+#  index_pdf_subscriptions_on_frequency  (frequency)
+#  index_pdf_subscriptions_on_title      (title)
+#  index_pdf_subscriptions_on_user_id    (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+class PdfSubscription < ApplicationRecord
+  FREQUENCIES = [
+    WEEKLY = 'weekly',
+    MONTHLY = 'monthly'
+  ]
+
+  TITLES = [
+    ADMIN_USAGE_SNAPSHOT_REPORT = 'admin_usage_snapshot_report'
+  ]
+
+  belongs_to :user
+
+  validates :frequency, presence: true, inclusion: { in: FREQUENCIES }
+  validates :title, presence: true, inclusion: { in: TITLES }
+  validates :user_id, presence: true
```

Comment:
It may actually be worth adding a validation for the presence of `filter_selections` because, somewhat deceptively, even a user who hasn't selected any specific filters (thus using the default) will have at least an array of `school_ids` along with `timeframe_start` and `timeframe_end` set for them.  So we should always have some filter value for these.

Reply:
I'll defer to you and @emilia-friedberg on this question.  Also, does it make sense to add a similar validation for `AdminReportFilterSelections`